src/components/AssetSearch/AssetList.tsx

```diff
@@ -19,7 +24,12 @@ type ItemData<T> = {
   handleClick: T
 }
 
+type EnrichAsset = Asset & { fiatAmount: string; cryptoAmount: string; marketCap: string }
```

Comment:
See comment. This could go away using e.g a `useMemo` call and no "enriched asset" structure.